mocks/routes/thc.js

```diff
@@ -0,0 +1,132 @@
+// Use this file only as a guide for first steps using routes. Delete it when you have added your own route files.
+// For a detailed explanation regarding each routes property, visit:
+// https://mocks-server.org/docs/usage/routes
+
+const humidities = [
+  { timestamp: 1, humidity: 30 },
+  { timestamp: 2, humidity: 40 },
+  { timestamp: 3, humidity: 50 },
+  { timestamp: 4, humidity: 42 },
+  { timestamp: 5, humidity: 38 },
+]
+
+const temperatures = [
+  { timestamp: 1, temperature: 10.1 },
+  { timestamp: 2, temperature: 12.3 },
+  { timestamp: 3, temperature: 14.8 },
+  { timestamp: 4, temperature: 27.2 },
+  { timestamp: 5, temperature: 29.2 },
+]
+
+const co2s = [
+  { timestamp: 1, co2: 36 },
+  { timestamp: 2, co2: 81 },
+  { timestamp: 3, co2: 30 },
+  { timestamp: 4, co2: 64 },
+  { timestamp: 5, co2: 76 },
+]
+
+module.exports = [
+  {
+    id: "get-temperature", // route id
+    url: "/api/temperature", // url in express format
+    method: "GET", // HTTP method
+    variants: [
+      {
+        id: "success",
+        type: "middleware", // variant of type "middleware"
+        options: {
+          middleware: (req, res) => {
+            const query = req.query 
+            if (query.current==true) {
```

Comment:
Checking for current should be done like this because if the request has the parameter current "***/api/temperature?current" should return true no matter the value and "***/api/temperature" should be false.
```suggestion
            if (query.hasOwnProperty("current")) {
```

Reply:
But what if the parameter is "?current=false"? We should return the temperature if and only if the current is set to true, and never if it's not set or set to false I think

Reply:
Well if you look into our interface contract that if current is set it should return current temperature. Does not matter what is it set to. If someone does not want current data the flag is omitted.